qiita_db/analysis.py

```diff
@@ -100,11 +100,14 @@ def get_by_status(cls, status):
         set of int
             All analyses in the database with the given status
         """
-        conn_handler = SQLConnectionHandler()
-        sql = ("SELECT analysis_id FROM qiita.{0} a JOIN qiita.{0}_status ans "
-               "ON a.analysis_status_id = ans.analysis_status_id WHERE "
-               "ans.status = %s".format(cls._table))
-        return {x[0] for x in conn_handler.execute_fetchall(sql, (status,))}
+        with TRN:
+            sql = """SELECT analysis_id
```

Comment:
@squirrelo can you point to the "standard" that you're talking about? There is no standard, only coding guidelines (that we can user or not) but None of the ones that I've seen use your indentation. Usually is common practice to put the FROM in the next line under the SELECT, not following it (quick googling will show this to you...)

I can agree on adding the JOIN in a single line but it didn't fit. Will change for USING.